cloud/cloud_scheduler.cc

```diff
@@ -187,6 +223,22 @@ long CloudSchedulerImpl::ScheduleRecurringJob(
   return id;
 }
 
+bool CloudSchedulerImpl::IsScheduled(long id) {
```

Comment:
Is the IsScheduled() part here needed to fix LITE and UNITY Tests?

Reply:
The IsScheduled was added to fix some TSAN race cases.  This method was to allow the tests to know whether or not the job still existed, rather than waiting a period of time and assuming it had completed.